lib/src/server/server.dart

```diff
@@ -166,7 +183,8 @@ class Server extends ConnectionServer {
   /// Create a server for the given [services].
   @Deprecated('use Server.create() instead')
   Server(
-    super.services, [
+    super.services,
+    super.keepAlive, [
```

Comment:
This is breaking change. Consider just not adding support for keepAlive here and intialize it with default value instead.